plus/src/test/resources/sql/cross-join-lateral.iq

```diff
@@ -39,7 +39,7 @@ CROSS JOIN LATERAL TABLE(AUX.CODES(SC.email)) TF limit 6;
 # Checks whether CROSS JOIN LATERAL WORK WITH VIEW EXPANSION
 SELECT * FROM EXAMPLES.CODED_EMAILS;
 +-----------------------+----------+------------------------------+
-| EMAIL                 | TYPE     | CODEVALUE                    |
+| email                 | TYPE     | CODEVALUE                    |
```

Comment:
Lex of of view sql of `EXAMPLES.CODED_EMAILS` is MYSQL and its definition is as below in chinook.json
```
SELECT SC.email, TF.TYPE, TF.CODEVALUE
FROM ...
```
After this change, Casing is kept/unchanged.